changelog.rst

```diff
@@ -0,0 +1,266 @@
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+[0.5.0] -- 2019-0x-xx
+---------------------
+Added
++++++
+  - Rowan is now a dependency.
+  - Add GETAR file reader/writer.
+  - Add ``shape_dict`` representation to ``Shape`` classes.
+  - Add support for particle properties:
+    - mass
+    - charge
+    - diameter
+    - image
+    - moment of inertia
+    - angular momentum
+  - Add support for reading/writing shapes in GSD via HOOMD-HPMC state.
+  - Add universal reader/writer with format detection.
+  - Add orientable attribute to spheres.
+  - Extend list of supported shape classes:
+    - ellipsoid
+    - polygon
+    - spheropolygon
+    - convex polyhedron
+    - convex spheropolyhedron
+
+Changed
++++++++
+  - Raise ``AttributeError`` if accessing an invalid frame or trajectory property.
+  - Rename several existing shape classes.
+  - Improve unit test coverage.
+  - Revise documentation.
+  - Move shape definitions to separate module.
+
+Deprecated
+++++++++++
+  - Tests for Python 2 are no longer updated (Python 2 support will be entirely dropped in a future release).
```

Comment:
I would recommend to be more precise here:

> will be dropped in the next minor release.

That will make it easier for you to feel comfortable dropping the support.